### PR TITLE
feat: Track Tissues and Genes in Image Download

### PR DESCRIPTION
**Pull request by dsadgat**
- #3486

## Changes
Added 'tissues' and 'genes' to analytics upon image download in WMG for better understanding of usefulness.

## QA steps
I printed out to console while locally testing - on image download, this is now what is being sent to plausible:
`WMG_DOWNLOAD_COMPLETE {file_type: 'png', tissues: 'blood,lung,spleen', genes: 'TSPAN6,TNMD,DPM1'}`

### CODE REVIEW

**Review by atarashansky — APPROVED** [review]

File: frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveImage/index.tsx [hunk 1]
```diff
@@ -108,7 +108,7 @@ export default function SaveImage({
       }
       link.click();
       link.remove();
-      track(EVENTS.WMG_DOWNLOAD_COMPLETE, { file_type: fileType });
+      track(EVENTS.WMG_DOWNLOAD_COMPLETE, { file_type: fileType , tissues: selectedTissues.toString(), genes: selectedGenes.toString() });
```
Comment (tihuan) [inline frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveImage/index.tsx:111]: Just double checking, did Amanda say if she wants us to send the tissue count and gene count for her as well?

Thank you!

**Review by tihuan — APPROVED** [review]
LGTM 🚀 ! Just one quick question regarding the payload 🙆‍♂️ Thanks, @dsadgat 🎉 !

File: frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveImage/index.tsx (hunk 1 above)
Comment (dsadgat) [reply frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveImage/index.tsx:111]: No she said this was all she needed :)

Comment (tihuan) [reply frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveImage/index.tsx:111]: Awesome! 

Last question: did she say anything about the order of the array? E.g., does she care about [A, B] vs. `[B, A]`? Just wondering if it'd be easier for her to identify the common tissues and genes combo if we sorted the arrays for her, or if she wants to preserve the user selected order or something 😄

Comment (dsadgat) [reply frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveImage/index.tsx:111]: Oh 🤔 I am under the assumption that they come in ordered by top to bottom, is that wrong? Do they come in by order they clicked?

Comment (tihuan) [reply frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveImage/index.tsx:111]: Ah yeah just tested and the order is by clicked!

Comment (dsadgat) [reply frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveImage/index.tsx:111]: Confirmed by clicked is good! Thanks for bringing that up!

**Review by seve — APPROVED** [review]

File: frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveImage/index.tsx (hunk 1 above)
Comment (tihuan) [reply frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveImage/index.tsx:111]: awesomee thanks so much for confirming, @dsadgat 🤩 🚀 !

**Pull request merged**